scripts/xsdata/process_j312.bash

```diff
@@ -17,29 +19,47 @@ TEMPS=(900)
 
 for T in ${TEMPS[@]}
 do
-    wget -P $DATADIR $LN$SLUG$T$EXT
+    if [[ ! -f $DATADIR/$SLUG$T$EXT ]]
+    then
+        wget -P $DATADIR $LN$SLUG$T$EXT
+    fi
 done
 
 # Get thermal XS
-wget -O $DATADIR/ACEs_THERMK.zip $LN$SLUG1
+if [[ ! -f $DATADIR/ACEs_THERMK.zip ]]
+then
+    wget -O $DATADIR/ACEs_THERMK.zip $LN$SLUG1
+fi
+
+
+# Get neutron induced and spontaneous fission yield data from JEFF 3.3
+if [[ ! -f $DATADIR/sss_jeff33.nfy ]]
+then
+    wget -O $DATADIR/sss_jeff33.nfy https://www.oecd-nea.org/dbdata/jeff/jeff33/downloads/JEFF33-nfy.asc
+fi
```

Comment:
Does this pull data from JEFF 3.3? I thought this was for getting data from JEFF 3.1.2 ???

Reply:
There is no spontaneous or delayed neutron fission data in JEFF 3.1.2, nor is there decay data. The data from the JEFF 3.3 library is backwards compatible with the JEFF 3.1.2 library.

Reply:
That makes sense. Just wanted to clarify the reason for the version change.